docs/root/intro/version_history.rst

```diff
@@ -7,6 +7,8 @@ Version history
   <arch_overview_load_balancing_types_round_robin>` support. The round robin
   scheduler now respects endpoint weights and also has improved fidelity across
   picks.
+* :ref:`Locality weighted load balancing <arch_overview_load_balancer_subsets>`
```

Comment:
Prior release notes are in the form 
component: description  
If we're sticking with that, can you update this and the round robin one to be prefaced with "loadbalancing:" ?

It'll also help with merge if we update components rather than just add things to the bottom